ios/FluentUI/Navigation/SearchBar.swift

```diff
@@ -574,6 +574,75 @@ open class SearchBar: UIView {
     private func dismissKeyboard() {
         searchTextField.resignFirstResponder()
     }
+
+    // MARK: - Cancel Button Accessibility Properties
+
+    public func setAllCancelButtonAccessibilityProperties(accessibilityHint: String?, accessibilityLabel: String?, accessibilityIdentifier: String?){
+        cancelButtonAccessibilityHint = accessibilityHint
+        cancelButtonAccessibilityLabel = accessibilityLabel
+        cancelButtonAccessibilityIdentifier = accessibilityIdentifier
+    }
+
+    public var cancelButtonAccessibilityHint: String? {
+        get {return cancelButton.accessibilityHint}
```

Comment:
I think we usually have spaces around the content, so `{ return cancelButton.accessibilityHint }`

Reply:
https://github.com/microsoft/fluentui-apple/runs/6700072532?check_suite_focus=true#step:5:146 the lint should help with this in the future! :)